app/app/src/main/res/layout/content_joystick.xml

```diff
@@ -15,35 +39,39 @@
         android:visibility="invisible" />
 
     <!-- Forward button -->
+
+    <!-- Backward button -->
+
+    <!-- Left button -->
+
+    <!-- Right button -->
+
     <ImageButton
```

Comment:
This button should not be included in the layout if there is a joystick to replace it.